LDAP/ldap2s3/profile.json

```diff
@@ -0,0 +1,31 @@
+{
+	"user_id": "",
+	"timezone": "",
+	"active": true,
```

Comment:
this defaults users to active as we don't know when users are inactive. the active vs inactive needs to be deduced upstream at this time... cc @andrewkrug